ibridges/data_operations.py

```diff
@@ -474,6 +498,16 @@ def _get_data_objects(session: Session,
 
     return objs
 
+def _get_subcoll_paths(session: Session,
+                     coll: irods.collection.iRODSCollection) -> list:
```

Comment:
This doesn't look properly lined up, but it could be GitHub.

Reply:
It just looks like that in github, on the terminal it looks ok and pylint does not complain.